node/src/db/db_handlers.rs

```diff
@@ -0,0 +1,1140 @@
+use std::{
+    collections::{HashMap, HashSet},
+    sync::Arc,
+};
+
+use crate::{
+    bead::Bead,
+    braid::{consensus_functions, Braid},
+    db::{
+        init_db::init_db, AncestorTimestampTuple, BeadTuple, BraidpoolDBTypes, CohortIdTuple,
+        CohortTuple, InsertTupleTypes, ParentTimestampTuple, RelativeTuple, TransactionTuple,
+    },
+    error::DBErrors,
+};
+use futures::lock::Mutex;
+use num::ToPrimitive;
+use sqlx::{Pool, Sqlite};
+use tokio::sync::{
+    mpsc::{Receiver, Sender},
+    RwLock,
+};
+
+#[derive(Debug)]
+pub struct DBHandler {
+    //Query reciever inherit to handler only
+    receiver: Receiver<BraidpoolDBTypes>,
+    //Shared across tasks for accessing DB after contention using `Mutex`
+    pub db_connection: Arc<Mutex<Pool<Sqlite>>>,
+    local_braid_arc: Arc<RwLock<Braid>>,
+}
+// The DATABASE_URL environment variable must be set at build time to a database which it can prepare queries against; the database does not have to contain any data but must be the same kind (MySQL, Postgres, etc.) and have the same schema as the database you will be connecting to at runtime.
+//A transaction starts with a call to Pool::begin or Connection::begin.
```

Comment:
This comment is now incorrect since we put the TRANSACTION statement in the SQL itself.